src/components/camera.js

```diff
@@ -1,6 +1,7 @@
 var registerComponent = require('../core/component').registerComponent;
 var THREE = require('../lib/three');
 var utils = require('../utils/');
+var bind = utils.function.bind;
```

Comment:
`function` resonates with you because you're familiarized with `underscore`. Without that context is meaningless.